app/config/config.yml

```diff
@@ -15,6 +15,8 @@ parameters:
     env(PS_LOG_OUTPUT): "%kernel.logs_dir%/%kernel.environment%.log"
     mail_themes_uri: "/mails/themes"
     mail_themes_dir: "%kernel.project_dir%%mail_themes_uri%"
+    prestashop_temporary_image_directory: "%kernel.project_dir%/img/tmp/"
```

Comment:
```suggestion
    prestashop_temporary_image_directory: "%kernel.project_dir%/var/tmp/"
```

Reply:
We already know we are in a prestashop context :)

Reply:
@PierreRambaud  @matks as we with @rokaszygmantas  see without vfstream using these injected parameters cause more problems / refactoring / creating non existing folder structures. So we suggest to not use these injected parameters now and use folder structures like we /img/tmp and img/c